native/chat/chat-thread-background.react.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import * as React from 'react';
+import ChatThreadList from './chat-thread-list.react';
+import type { ChatTopTabsNavigationProp } from './chat.react';
+import { threadInBackgroundChatList } from 'lib/shared/thread-utils';
+import { type ThreadInfo, type RawThreadInfo } from 'lib/types/thread-types';
+
+type BackgroundChatThreadListProps = {
+  navigation: ChatTopTabsNavigationProp<'BackgroundChatThreadList'>,
+};
+export default function BackgroundChatThreadList(
+  props: BackgroundChatThreadListProps,
+) {
+  const filterFunction = React.useCallback(
+    (item: ?(ThreadInfo | RawThreadInfo)) => {
+      return threadInBackgroundChatList(item);
+    },
+    [],
+  );
+  return (
```

Comment:
(We still need to do the `setOptions` call to set the title so it includes the unread count, but actually this would be better in a separate PR)

Reply:
Yes, I'm working on that, just didn't want to put everything in one big PR